msb/rawlogger/msb_rawlogger.py

```diff
@@ -15,15 +15,16 @@ def signal_handler(sig, frame):
 
 
 class RawLoggerService:
-    def __init__(self, config: RawLoggerConf, subscriber: ZMQRawSubscriber):
+    def __init__(self, config: RawLoggerConf, subscriber: ZMQRawSubscriber, excluded_topics):
+        self.excluded_topics = excluded_topics
         self.config = config
         self.subscriber = subscriber
         self.logger = RawLogger(config)
-
+        print(self.logger,"self.logger")
         if self.config.verbose:
             print(self.config)
 
-    def get_data(self) -> tuple[bytes, bytes]:
+    def get_data(self,excluded_topics) -> tuple[bytes, bytes]:
```

Comment:
Do not pass `excluded_topics` here, use `self.excluded_topics` if necessary, but it does seem to be used here.

Reply:
Removed excluded_topics from get_data method